src/test/java/ExtensionIT.java

```diff
@@ -0,0 +1,185 @@
+    // - (SDK action listener) test action listener work
+
+    private int port;
+    private Settings settings;
+    private final int minPort = 49152;
+    private final int maxPort = 65535;
+    private final String host = "127.0.0.1";
+    private volatile String clientResult;
+
+    @BeforeEach
+    public void setUp() {
+
+        // randomize port selection when creating settings
+        port = getRandomPort();
+
+        settings = Settings.builder()
+            .put("node.name", "node_extension_test")
+            .put(TransportSettings.BIND_HOST.getKey(), host)
+            .put(TransportSettings.PORT.getKey(), port)
+            .build();
+    }
+
+    @Test
+    public void testThatInfosAreExposed() {
+
+        RunPlugin runPlugin = new RunPlugin();
+        ThreadPool threadPool = new TestThreadPool("test");
+        Netty4Transport transport = runPlugin.getNetty4Transport(settings, threadPool);
+
+        // start netty transport and ensure that address info is exposed
+        try {
+            transport.start();
+            assertEquals(transport.lifecycleState(), Lifecycle.State.STARTED);
+
+            // check bound addresses
+            for (TransportAddress transportAddress : transport.boundAddress().boundAddresses()) {
+                assert (transportAddress instanceof TransportAddress);
+                assertEquals(host, transportAddress.getAddress());
+                assertEquals(port, transportAddress.getPort());
+            }
+
+            // check publish addresses
+            assert (transport.boundAddress().publishAddress() instanceof TransportAddress);
+            TransportAddress publishAddress = transport.boundAddress().publishAddress();
+            assertEquals(host, NetworkAddress.format(publishAddress.address().getAddress()));
+            assertEquals(port, publishAddress.address().getPort());
+
+        } finally {
+            terminate(threadPool);
+        }
+    }
+
+    @Test
+    public void testInvalidMessageFormat() throws UnknownHostException, InterruptedException {
+        Thread client = new Thread() {
+            @Override
+            public void run() {
+                try {
+
+                    // Connect to the server
+                    Socket socket = new Socket(host, port);
+
+                    // Create input/output stream to read/write to server
+                    BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+                    PrintStream out = new PrintStream(socket.getOutputStream());
+
+                    // server logs MESSAGE RECIEVED: TESTTT
+                    // note : message validation is only done if message length >= 6 bytes
+                    // character is 1 byte
+                    out.print("TESTTT");
+
+                    // Exception will originate from org.opensearch.transport.TcpTransport
```

Comment:
Change the single line comments to multie line comments if comments has multiple lines here and for the rest

Reply:
I see you have removed the comment. It's better to have comments, just change it to multi line comment